web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataService.kt

```diff
@@ -101,7 +109,8 @@ class PersondataServiceImpl(
             tredjepartspersonMapper.lagTredjepartsperson(
```

Comment:
[out-of-scope] denne funksjonen kan vi skrive om til å ta ibruk den nye pdl-operasjonen: `pdl.hentAdressebeskyttelse`. Så slipper vi å hente mer enn nødvendig informasjon her.